command/txpool/txpool.go

```diff
@@ -23,8 +22,8 @@ func GetCommand() *cobra.Command {
 
 func registerSubcommands(baseCmd *cobra.Command) {
 	baseCmd.AddCommand(
-		// txpool add
-		add.GetCommand(),
+		// txpool add - removed
+		// we don't support adding unsigned transactions
```

Comment:
You can completely remove these two lines

Reply:
Removed in 44aa6d3b2da32c187d9100d5abda802d9164d8ae